keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -2641,42 +2705,36 @@ when the feature is disabled.
 
 ### Rollout, Upgrade and Rollback Planning
 
-<!--
-This section must be completed when targeting beta to a release.
--->
-
 ###### How can a rollout or rollback fail? Can it impact already running workloads?
 
-<!--
-Try to be as paranoid as possible - e.g., what if some components will restart
-mid-rollout?
+Workloads not using ResourceClaims should not be impacted because the new code
+will not do anything besides checking the Pod for ResourceClaims.
 
-Be sure to consider highly-available clusters, where, for example,
-feature flags will be enabled on some API servers and not others during the
-rollout. Similarly, consider large clusters and how enablement/disablement
-will rollout across nodes.
--->
+When kube-controller-manager fails to create ResourceClaims from
+ResourceClaimTemplates, those Pods will not get scheduled. Bugs in
+kube-scheduler might lead to not scheduling Pods that could run or worse,
+schedule Pods that should not run. Those then will get stuck on a node where
+kubelet will refuse to start them. None of these scenarios affect already
+running workloads.
+
+Failures in kubelet might affect running workloads, but only if containers for
+those workloads need to be restarted.
```

Comment:
I know that the likehold of this is low but what do we do about existing workloads that use resource claims?

Will bumping this from alpha to beta work for existing workloads? I guess they would be already scheduled?

Reply:
Upgrades will work seamlessly. All resources using v1alpha3 will get converted to v1beta1 when reading or writing them.